components/sponsored_benefits/app/views/sponsored_benefits/organizations/plan_design_proposals/edit.html.erb

```diff
@@ -3,4 +3,21 @@
   <%= form_for @plan_design_proposal, url: {action: "update"}, method: :put do |f| %>
      <%= render 'form', f: f %>
   <% end %>
+
+
+  <%= render 'employees' %>
+
+  <br />
+  <div class="row">
+    <div class="col-md-12">
+
+      <div class="pull-left">
+        <%= link_to 'Return to Quote Management', sponsored_benefits.organizations_plan_design_organization_plan_design_proposals_path(@plan_design_proposal.plan_design_organization), class:'btn btn-primary' %>
+      </div>
+
+      <div class="pull-right">
+        <%= link_to "Select Health Benefits", new_organizations_plan_design_proposal_plan_selection_path(@plan_design_proposal.proposal.id), remote: true, class:'btn btn-primary' %>
```

Comment:
Please check this commit- cb577d359691db69ca6c9478707cad37f1a78a85

Reply:
Thanks for the catch @sri49.